src/DashboardService.Data.Provider.MsSql.Ef/Migrations/20231211221100_InitialCreate.cs

```diff
@@ -0,0 +1,101 @@
+using LT.DigitalOffice.DashboardService.Models.Db;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+
+namespace LT.DigitalOffice.DashboardService.Data.Provider.MsSql.Ef.Migrations;
+
+[DbContext(typeof(DashboardServiceDbContext))]
+[Migration("20220729014055_InitialCreate")]
+public class InitialCreate : Migration 
+{
+  protected override void Up(MigrationBuilder migrationBuilder)
+  {
+    CreateTaskTypeTable(migrationBuilder);
+    CreatePriorityTable(migrationBuilder);
+    CreateTaskTable(migrationBuilder);
+  }
+  
+  protected override void Down(MigrationBuilder migrationBuilder)
+  {
+    migrationBuilder.DropTable(DbTask.ToTable);
+    migrationBuilder.DropTable(DbPriority.ToTable);
+    migrationBuilder.DropTable(DbTaskType.ToTable);
+  }
+
+  private void CreateTaskTypeTable(MigrationBuilder migrationBuilder)
+  {
+    migrationBuilder.CreateTable(
+      name: DbTaskType.ToTable,
+      columns: table => new
+      {
+        Id = table.Column<Guid>(nullable: false),
+        Name = table.Column<string>(nullable: false, type: "nvarchar(50)", maxLength:50),
+      },
+      constraints: table =>
+      {
+        table.PrimaryKey($"PK_{DbTaskType.ToTable}", x => x.Id);
+      }
+    );
+  }
+
+  private void CreatePriorityTable(MigrationBuilder migrationBuilder)
+  {
+    migrationBuilder.CreateTable(
+      name: DbPriority.ToTable,
+      columns: table => new
+      {
+        Id = table.Column<Guid>(nullable: false),
+        Name = table.Column<string>(nullable: false, type: "nvarchar(50)", maxLength:50),
+      },
+      constraints: table =>
+      {
+        table.PrimaryKey($"PK_{DbPriority.ToTable}", x => x.Id);
+      }
+    );
+  }
+  
+  private void CreateTaskTable(MigrationBuilder migrationBuilder)
+  {
+    migrationBuilder.CreateTable(
+      name: DbTask.ToTable,
+      columns: table => new
+      {
+        Id = table.Column<Guid>(nullable: false),
+        GroupId = table.Column<Guid>(nullable: false),
+        TaskTypeId = table.Column<Guid>(nullable: true),
+        PriorityId = table.Column<Guid>(nullable: true),
+        Name = table.Column<string>(nullable: false, type: "nvarchar(50)", maxLength:50),
+        Content = table.Column<string>(nullable: false, type: "nvarchar(max)"),
+        CreatedAtUtc = table.Column<DateTime>(nullable: false),
+        DeadLineAtUtc = table.Column<DateTime>(nullable: true),
```

Comment:
May be  Dead**l**ine?

Reply:
Actually it has sense, maybe we should use it too